aws-lambda/massdriver.yaml

```diff
@@ -7,27 +7,80 @@ type: application
 
 params:
   required:
-  - region
-  - port
-  - log_level
```

Comment:
We’ve started calling this ‘region’ or $resourceType.region (database.region) in newer bundles